cheroot/__main__.py

```diff
@@ -0,0 +1,103 @@
+import argparse
+from importlib import import_module
+import os
+import sys
+
+import six
+
+from .wsgi import Server
+
+
+def wsgi_app_path(string):
+    if ':' in string:
+        mod_path, app_path = string.split(':', 1)
+    else:
+        mod_path, app_path = string, 'application'
+
+    return mod_path, app_path
+
+
+def wsgi_bind_addr(string):
+    if ':' in string:
+        addr, port = string.split(':', 1)
+        port = int(port)
+    else:
+        addr, port = string, 8000
+    return addr, port
```

Comment:
@jaraco I've been thinking about having a public struct for the interface in our codebase. What do you think?

It could be:
1. reused across the project
2. support several different concrete socket implementations (and maybe hold some list of features they have):
  - TCP sockets
  - UNIX file sockets (most unix-like OSs)
  - UNIX abstract namespace sockets (GNU/Linux starting v2.2.0 of Linux kernel)